pkgs/games/fish-fillets-ng/default.nix

```diff
@@ -13,8 +13,72 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ pkgconfig ];
   buildInputs = [SDL lua5_1 SDL_mixer SDL_image SDL_ttf];
   postInstall=''
+    # Install game data
     mkdir -p "$out/share/games/fillets-ng/"
     tar -xf ${data} -C "$out/share/games/fillets-ng/" --strip-components=1
+    
+    # Install desktop file
+    mkdir -p $out/share/applications
+    cat > $out/share/applications/fish-fillets-ng.desktop <<EOF
+    [Desktop Entry]
+    Name=Fish Fillets
+    Name[bg]=Рибни филета
+    Comment=Puzzle game about witty fish saving the world sokoban-style
+    Comment[bg]=Логическа игра, в която духовити рибки спасяват света в стил Сокобан
+    Comment[de]=Puzzlespiel mit witzigen Fischen, die die Welt im Sokoban-Stil retten
+    Comment[ro]=Un puzzle cu un pește inteligent care salvează lumea în stil sokoban
+    Exec=$out/bin/fillets
+    Icon=fish-fillets-ng
+    Terminal=false
+    Type=Application
+    Categories=Game;LogicGame;
+    Keywords=sokoban;puzzle;
+    Keywords[bg]=сокобан;ребус;
+    EOF
+
+    # Install game icon
+    mkdir -p $out/share/pixmaps
+    cat > $out/share/pixmaps/fish-fillets-ng.xpm <<'EOF'
```

Comment:
This is just a static file, maybe we should fetch it or at least commit separately?

Reply:
IIRC there is a function for this which should be used.

Reply:
@7c6f434c Good idea. Back then I was (unjustifiedly) scared, not knowing offhand how it would work. But of course it's rather easy, the new force-push has the icon as a separate file.

@SuperSandro2000 Indeed, thanks, I didn't know it back then but do know. The new force-push uses makeDesktopItem and copyDesktopItems. :-)